src/pages/OfferNew/Perils/types.ts

```diff
@@ -8,3 +8,25 @@ export interface Peril {
   info?: string
   icon: JSX.Element
 }
+
+type IconVariant = {
+  svgUrl: string
+}
+
+type IconVariants = {
+  dark: IconVariant
+  light: IconVariant
+}
+
+export type PerilIcon = {
+  variants: IconVariants
+}
+
+export interface PerilV2 {
+  title: React.ReactNode
+  description: string
+  covered: string[]
+  exceptions: string[]
+  info?: string
+  icon: PerilIcon
+}
```

Comment:
Correct me if i'm wrong but aren't these types in the graphql schema already? If so we could use them directly from `data/graphql` after running `yarn gen`

Reply:
Ah, of course! 😄